apps/molgenis-components/src/components/tables/cellTypes/FileDisplay.vue

```diff
@@ -1,7 +1,5 @@
 <template>
-  <a v-if="data.id" :href="data.url">
-    {{ metaData.name }}.{{ data.extension }} ({{ fileSize }})
-  </a>
+  <a v-if="data.id" :href="data.url"> {{ data.filename }} ({{ fileSize }}) </a>
```

Comment:
I think we should change to {{ date.filename ? data.filename : metaData.name+'.'+data.extension }} because the filename will be empty for data that was uploaded before this filename option was there?